File: src/main/java/team4/footwithme/member/service/response/LoginResponse.java
package team4.footwithme.member.service.response;

import team4.footwithme.member.jwt.response.TokenResponse;

public record LoginResponse(
        String accessToken,
        String refreshToken
) {
    public static LoginResponse from(TokenResponse tokenResponse){
        return new LoginResponse(tokenResponse.accessToken(), tokenResponse.refreshToken());
    }
}

File: src/main/java/team4/footwithme/member/service/response/LoginResponse.kt
package team4.footwithme.member.service.response

import team4.footwithme.member.jwt.response.TokenResponse

@JvmRecord
data class LoginResponse(
    @JvmField val accessToken: String?,
    @JvmField val refreshToken: String?
) {
    companion object {
        fun from(tokenResponse: TokenResponse?): LoginResponse {
            return LoginResponse(tokenResponse!!.accessToken, tokenResponse.refreshToken)
        }
    }
}
